core/node/rpc/archiver.go

```diff
@@ -403,14 +468,34 @@ func (a *Archiver) ArchiveStream(ctx context.Context, stream *ArchiveStream) err
 		a.miniblocksProcessed.Add(uint64(len(serialized)))
 	}
 
-	// All blocks processed, mark stream as current
-	stream.stale.Store(false)
-
 	return nil
 }
 
+func (a *Archiver) emitPeriodicCorruptStreamReport(ctx context.Context) {
```

Comment:
the debug endpoints will not be available until Kerem gets back in the office and sets it up in AWS, so for now let's periodically print a report to logs